bin/v2.js

```diff
@@ -474,59 +542,86 @@ const configureCmd = (gurglerPath, bucketPath, commit, branch) => {
  * @param globs
  * @param pretend {boolean}
  */
-const deployCmd = (bucketNames, gurglerPath, globs, pretend= false) => {
-  fs.readFile(gurglerPath, 'utf-8', (err, data) => {
+const deployCmd = (bucketNames, gurglerPath, globs, pretend = false) => {
+  fs.readFile(gurglerPath, "utf-8", (err, data) => {
     if (err) {
-      if (err.code === 'ENOENT') {
-        console.log("Either gurgler.json has not been built or it has been removed. Run 'gurgler configure <git commit sha> <git branch>' and try again.");
+      if (err.code === "ENOENT") {
+        console.log(
+          "Either gurgler.json has not been built or it has been removed. Run 'gurgler configure <git commit sha> <git branch>' and try again."
+        );
       }
       throw err;
     }
 
     let localFilePaths = [gurglerPath];
 
-    globs.forEach(globb => {
-      localFilePaths = localFilePaths.concat(glob.sync(globb.pattern, {
-        ignore: globb.ignore
-      }))
-    })
+    globs.forEach((globb) => {
+      localFilePaths = localFilePaths.concat(
+        glob.sync(globb.pattern, {
+          ignore: globb.ignore,
+        })
+      );
+    });
 
-    const {prefix, raw} = JSON.parse(data)
+    const { prefix, raw } = JSON.parse(data);
 
-    localFilePaths.forEach(localFilePath => {
-      readFileAndDeploy(bucketNames, prefix, localFilePath, raw, pretend);
+    localFilePaths.forEach((localFilePath) => {
+      if (fs.statSync(localFilePath).isFile()) {
```

Comment:
For anyone going back through this PR - this is where the main change is. Everything else is eslint.